src/main/kotlin/blackjack/model/Player.kt

```diff
@@ -0,0 +1,25 @@
+package blackjack.model
+
+data class Player(
+    val name: String,
+) {
+    val cards: Cards = initCards()
+
+    init {
+        validateName()
+    }
+
+    private fun validateName() {
+        if (!name.matches(Regex("^[A-Za-z]+$"))) throw IllegalArgumentException("이름이 잘못 입력되었습니다.")
```

Comment:
해당 정규표현식 객체도 매번 함수 호출시마다 생성되기에는 비효율적인 것 같은데 상수로 만들어두면 어떨까요?

그리고 이런류의 예외검사라면 다음과 같이도 할 수 있습니다 ㅎㅎ 

```suggestion
require (name.matches(Regex("^[A-Za-z]+$"))) {
    "이름이 잘못 입력되었습니다."
}
```

Reply:
헛... 넵! 수정해두겠습니다!!